### PR TITLE
Bundle packages

### PR DESCRIPTION
**Pull request by paescuj**
Enable straightforward bundling of the following packages:
- `@directus/composables`
- `@directus/constants`
- `@directus/pressure`
- `@directus/schema`
- `@directus/storage-driver-azure`
- `@directus/storage-driver-cloudinary`
- `@directus/storage-driver-gcs`
- `@directus/storage-driver-local`
- `@directus/storage-driver-s3`
- `@directus/storage-driver-supabase`
- `@directus/storage`
- `@directus/update-check`

Also align `tsconfig.json` files:
- Remove `outDir` which is no longer necessary
- Ensure `include` is defined to prevent warnings like
  > Cannot write file 'packages/data-sql/dist/index.d.ts' because it would overwrite input file

### CODE REVIEW

**Review by jaads — APPROVED** [review]
Looks all good to me. bundles as expected 🎉

File: packages/schema/package.json [hunk 1]
```diff
@@ -25,24 +25,22 @@
 	"exports": {
 		".": "./dist/index.js",
 		"./package.json": "./package.json",
-		"./types/overview": {
-			"types": "./dist/types/overview.d.ts",
-			"default": "./dist/types/overview.js"
-		}
+		"./types/overview": "./dist/types/overview.js"
 	},
```
Comment (br41nslug) [inline packages/schema/package.json:29]: why are we explicitly bundling `./types/overview.js` separately here? Should we not export this from the root too? 🤔

**Review by br41nslug — COMMENTED** [review]
Considering you have almost all storage drivers included do we want to bundle `@directus/storage-driver-supabase` too in this PR?

**Comment by paescuj** [comment]
> Considering you have almost all storage drivers included do we want to bundle `@directus/storage-driver-supabase` too in this PR?

Hehe thanks! I guess this driver was added after I've created the original PR 😄

File: packages/schema/package.json (hunk 1 above)
Comment (paescuj) [reply packages/schema/package.json]: Good catch! This is a legacy thing. Although unrelated, I opted to clean that up in this PR 👍

**Review by azrikahar — APPROVED** [review]
LGTM! The built bundle runs without any noticeable issues on my end 👍

**Pull request merged**